python/lsst/ts/watcher/alarm.py

```diff
@@ -150,21 +157,57 @@ def configure(
         self.callback = callback
         self.auto_acknowledge_delay = auto_acknowledge_delay
         self.auto_unacknowledge_delay = auto_unacknowledge_delay
-        self.escalate_to = escalate_to
-        self.escalate_delay = escalate_delay
+
+    def configure_escalation(self, escalation_delay, escalation_responders):
+        """Configure escalation.
+
+        Set the following attributes:
+
+        * escalation_delay
+        * escalation_responders
+        * escalation_responders_json
+
+        Parameters
+        ----------
+        escalation_delay : `float`
+            Delay before escalating a critical unacknowledged alarm (sec).
+            If 0 the alarm is not escalated.
+        escalation_responders : `list`
+            Who or what to escalate the alarm, as a sequence of
+            ``{"name": "...", "type": "..."}``.
+            If an empty sequence the alarm will not be escalated.
+
+        Raises
+        ------
+        ValueError
+            If escalation_delay < 0.
+            If escalation_delay > 0 and escalation_responders empty,
+            or escalation_delay = 0 and escalation_responders not empty.
+        """
+        if escalation_delay < 0:
+            raise ValueError(f"{escalation_delay=} must be ≥ 0")
+        if (escalation_delay == 0) != (len(escalation_responders) == 0):
+            raise ValueError(
+                f"{escalation_delay=} must be > 0 if and only if"
+                f"{escalation_responders=} is not empty"
+            )
+        self.escalation_responders = list(escalation_responders)
```

Comment:
Why is an explicit cast to `list` needed here if the doc string for `escalation_responders` already indicates that it is a list?

Reply:
I don't try for fully correct types in doc strings (only in formal Python type annotations). list is the shortcut I always for any generic ordered sequence.